### ISSUE TITLE
Codify Silences involving User-level GitOps

### ISSUE THREAD

**Issue by wmhutchison** [bcgov/developer-experience#4728]
**Describe the issue**
Review current silences involving user GitOps issues that Platform Operations do not need to be told about, and codify the silences so that the web console entries can be removed/reduced.

**Additional context**
Add any other context, attachments or screenshots

**How does this benefit the users of our platform?**

**Definition of done**
- [ ] Create/review manual proof of concept silence aiming to minimize the number of silences required.
- [ ] Codify via PR and test in a LAB cluster.
- [ ] Apply to the rest of the clusters once PR is approved and merged.

**Comment by wmhutchison** [comment]
Still  can exist, though the bulk of what is intended here will be covered by another ticket already in play (weekly silence reports for AlertManager).

Over the course of 2024 for OCP maintenance a closer look was taken at what actually is set off for things like OCP upgrades, and unfortunately most of what's triggered are either things we don't want to ever silence unless for a good reason (etcd degradation for example), or are triggering for things which we cannot easily isolate (stating a daemonset is degraded due to node maintenance, but nothing further than that).

The single valid use case for some automation for silence handling I can see is if we pursue an "oldie" from our OCP 3.11 days where we look at AAP handling via playbook the firmware upgrade via cordoing a batch of nodes, draining one at a time, and then booting off of virtual CD-ROM or the OneView equivalent. For such a case there are a small handful of AlertManager notifications normally smapping on-call that could be automated as part of this maintenance.

There are no hooks in OCP 4 for its own OCP upgrade automation, so the most common use case we cannot actually automate silences unless we're heavy-handed with the silences (not recommended). 

The most likely result from this ticket will be creating/improving documentation on how team members can properly use `amtooil` for various situations (apply the same silence on all clusters).

**Comment by wmhutchison** [comment]
Closing, replaced with https://citz-do.atlassian.net/browse/PLAT-159.
